.pre-commit-config.yaml

```diff
@@ -1,13 +1,9 @@
 repos:
 -   repo: https://github.com/ambv/black
-    rev: stable
+    rev: 20.8b1
     hooks:
     - id: black
-      name: black
-      language: system
-      entry: black
-      types: [python]
-      language_version: python3.6
+      language_version: python3
```

Comment:
Why this change from explicit python version into generic python3?

Reply:
I can put it back, it was useful until I realized I'd need to have more than one python around.